active/af6-bort.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime, timedelta
+
+from airflow.operators.python_operator import PythonOperator
+from airflow.models import DAG
+from operators.bort import generate_job_id, populate_blobid_in_job_table, run_bort_task
```

Comment:
this reminds me: I am going to add a TODO in the backlog to unify our generate_job_id/populate_job_id fn's into a single module that everything can call.